stack/mobile/redux.md

```diff
@@ -0,0 +1,8 @@
+# Redux
+
+En React y React Native se suele utilizar el concepto de store. Este es un estado global de la aplicación en el que se mantienen datos que se ocupan en varios lugares y por lo tanto no pertenecen al estado de un solo componente. Para esto ocupamos [Redux](https://redux.js.org/), posiblemente la herramienta más conocida y utilizada para esto.
+
+Hay algunas herramientas complementarias que probablemente necesites:
+
+* [Redux toolkit](https://redux-toolkit.js.org/): "redux con las baterias incluidas", este paquete incluye varias funciones para agilizar el desarrollo con Redux. Incluye funcionalidades que cumplen tareas muy comunes como configurar el store, crear una accion para cada reducer, o separar tu store en convenientes slices.
```

Comment:
- *acción
- En la palabra "slices" estaría bueno agregar este link: https://redux-toolkit.js.org/usage/usage-guide#creating-slices-of-state. Es como un concepto "nuevo" si no cachas el toolkit